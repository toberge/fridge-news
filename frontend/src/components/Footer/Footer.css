footer {
  background-color: var(--section-background);
  padding: 1rem 0;
  color: var(--section-text);
}

footer p {
  text-align: center;
}

footer ul {
  margin: 0 20%;
}
