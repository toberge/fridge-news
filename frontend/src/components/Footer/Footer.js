// @flow

import * as React from 'react';
import { Component } from 'react-simplified';

export default class Footer extends Component {
  render() {
    return (
      <footer>
        <p>Copyright &copy; 2019 Fridge News</p>
        <p>
          Logo credit: Icon made by <a href="https://www.freepik.com/home">Freepik</a> from{' '}
          <a href="http://www.flaticon.com/">www.flaticon.com</a>
        </p>
        <p>
          Current placeholder is public domain and does not require contribution.
          <br />
          Regardless, I found it on <a href="https://www.pexels.com/photo/office-disk-storage-data-41290/">
            Pexels
          </a>{' '}
          and it seems to be from{' '}
          <a href="https://www.publicdomainpictures.net/en/view-image.php?image=14548">Public Domain Pictures</a>
        </p>
        <ul>
          <li>
            <a href="mailto:contact@fridge.news" target="_top">
              Email
            </a>
          </li>
        </ul>
      </footer>
    );
  }
}
