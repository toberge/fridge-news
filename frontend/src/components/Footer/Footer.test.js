// @flow

import * as React from 'react';
import Footer from './Footer';
import { shallow, mount, ShallowWrapper, render } from 'enzyme';

describe('Footer', () => {
  const footer: ShallowWrapper = shallow(<Footer />);

  it('should render as intended', () => {
    // basically testing how snapshots are written
    expect(footer.debug()).toMatchSnapshot();
  });
});
