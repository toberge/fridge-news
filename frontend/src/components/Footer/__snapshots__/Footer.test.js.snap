// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render as intended 1`] = `
"<footer>
  <p>
    Copyright Â© 2019 Fridge News
  </p>
  <p>
    <EmailIcon />
     Send us a mail at
     
    <a href=\\"mailto:fridge.news@gmail.com\\" target=\\"_top\\">
      fridge.news@gmail.com
    </a>
  </p>
  <p>
    Current placeholder image is public domain and does not require contribution.
    <br />
    Regardless, I found it on 
    <a href=\\"https://www.pexels.com/photo/office-disk-storage-data-41290/\\">
      Pexels
    </a>
     
    and it seems to be from
     
    <a href=\\"https://www.publicdomainpictures.net/en/view-image.php?image=14548\\">
      Public Domain Pictures
    </a>
  </p>
</footer>"
`;
